drafts/draft-ietf-ippm-ioam-data-integrity.xml

```diff
@@ -791,22 +840,27 @@
         this document. Before that, the decapsulating node updates the ICV field
         in the IOAM Integrity Protection header. The Integrity Check Value (ICV)
         is the result of a GMAC operation over the received ICV field and
-        immutable IOAM-Data-Fields added by the decapsulating node. With the
-        received Nonce field provided to GMAC, the decapsulating node performs
-        the following operation:
+        immutable IOAM-Data-Fields added by the decapsulating node. In the case
+        of IOAM-Data-Fields pre-allocated by the encapsulating node, only those
```

Comment:
See above - please refer pre-allocated IOAM trace Option-Type instead of "IOAM-Data-Fields pre-allocated by the encap node".